scripts/test_pr_convergence.py

```
@@ -0,0 +1,990 @@
+            "data": {
+                "repository": {
+                    "pullRequests": {
+                        "nodes": [],
+                        "pageInfo": {"hasNextPage": True, "endCursor": "cursor-1"},
+                    }
+                }
+            }
+        }
+
+        with mock.patch.object(client, "run", return_value=response):
+            with self.assertRaises(pr_convergence.GhReadError):
```

Ruff (0.16.0)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)